docs/source/setup/ssh.rst

```diff
@@ -24,3 +24,50 @@ Or you can specify a hostfile that includes a list of hosts::
 The ``dask-ssh`` utility depends on the ``paramiko``::
 
     pip install paramiko
+
+
+CLI Options
+-------- 
+
+Launch a distributed cluster over SSH. A 'dask-scheduler' process will run
+on the first host specified in [HOSTNAMES] or in the hostfile (unless
+--scheduler is specified explicitly). One or more 'dask-worker' processes
+will be run each host in [HOSTNAMES] or in the hostfile. Use command line
+flags to adjust how many dask-worker process are run on each host
+(--nprocs) and how many cpus are used by each dask-worker process
+(--nthreads).
```

Comment:
We might also want to include the keywords in code highlighting

    ``--nthreads``